### PR TITLE
[FEAT] Heat Map v1 

### PR DESCRIPTION
**Pull request by jchen006**
Screen Shot 2020-06-02 at 12 01 18 AM

Created a heat map based on relative percentage so `countryTotalConfirmed/worldTotalConfirmed * 100`. 
I do want to add different options later down the road such as by population and be recovered and what not. 
Also need feedback on the actual tooltip itself since right now it's based on selecting a country and hovering over it. 
And yes, US and Brazil and Russia are getting hammered with cases ATM.

### CODE REVIEW

File: src/components/Maps/ChoroplethMap.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    Sphere,
+    Graticule
+} from 'react-simple-maps';
+
+function ChoroplethMap(props) {
+    const { 
+        geoUrl,
+        data,
+        geoMapFn
+    } = props
```
Comment (sbussard) [inline src/components/Maps/ChoroplethMap.jsx:14]: Is there any advantage to using a "props" variable vs destructuring the arguments directly?

```suggestion
function ChoroplethMap({ 
        geoUrl,
        data,
        geoMapFn
    }) {
```

File: src/features/world-choropleth-map/index.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import { useAsync } from 'react-use';
+import { ChoroplethMap } from '../../components/Maps/ChoroplethMap';
+import { Geography } from 'react-simple-maps';
+import Tooltip from '@material-ui/core/Tooltip';
+import { ChloroplethTooltip } from '../../components/Tooltip/ChloroplethTooltip';
+
+const geoUrl = 'https://raw.githubusercontent.com/zcreativelabs/react-simple-maps/master/topojson-maps/world-110m.json';
+
+function WorldChorolpethMap() {
+    const { value, error, loading } = useAsync(async () => {
```
[the remark this answers is not in the document]
Comment (dleng16) [reply src/features/world-choropleth-map/index.jsx:14]: Still getting familiar with react hooks & hoping for confirmation.
From my understanding using 'useAsync' is good practice over useState calls for value, error, loading. 
It's a 'custom react hook' from the react-use library?

File: src/components/Maps/ChoroplethMap.jsx (hunk 1 above)
Comment (jchen006) [reply src/components/Maps/ChoroplethMap.jsx:14]: @sbussard I think this is more of a stylistic thing and I personally find destructuring props in the argument can get pretty grotesque at some point especially for React but more than happen to make the update.

File: src/features/world-choropleth-map/index.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (jchen006) [reply src/features/world-choropleth-map/index.jsx:14]: @sbussard `react-use` has a lot of really cool hooks you should take a look! 
@dleng16 yeah it's a custom hook for from react-use library, I think the reason I'm using `useAsync` is to give a lot more control over how we want the API call to look. We could always build our own custom ones if there's going to be a lot of parameter passing down the road but this one is just a quick out of the box usage that gives three states to manage.

**Pull request merged**